src/napari/layers/shapes/_shapes_key_bindings.py

```diff
@@ -164,14 +165,14 @@ def paste_shape(layer: Shapes) -> None:
 def select_all_shapes(layer: Shapes) -> None:
     """Select/Deselect all shapes in the current view slice."""
     if layer._mode in (Mode.DIRECT, Mode.SELECT):
-        new_selected = set(np.nonzero(layer._data_view._displayed)[0])
+        new_selected = Selection(np.nonzero(layer._data_view._displayed)[0])
```

Comment:
`Selection` is an evented container, and it is slower than a pure `set`. I do not see a reason to use Selection here.

Reply:
I thought at first that it was this line was what triggered the mypy test to fail but am just now realizing that it was slightly below that. Will change back!